src/openklant/components/klantinteracties/api/serializers/internetaken.py

```diff
@@ -70,7 +85,52 @@ class Meta:
             },
         }
 
+    def to_representation(self, instance):
+        response = super().to_representation(instance)
+        response["toegewezen_aan_actor"] = ActorForeignKeySerializer(
+            instance.actoren.order_by("interneactorenthoughmodel__order").first(),
+            context={**self.context},
+        ).data
+
+        response["toegewezen_aan_actoren"] = ActorForeignKeySerializer(
+            instance.actoren.all().order_by("interneactorenthoughmodel__order"),
+            context={**self.context},
+            many=True,
+        ).data
+        return response
```

Comment:
Is this code necessary? It looks like it duplicates the code with field declaration

Reply:
It unfortunately is necessary. The issue is very well described in the Django Ordered Model library:
https://github.com/django-ordered-model/django-ordered-model?tab=readme-ov-file#ordering-of-manytomany-relationship-query-results